permits/geoservices.py

```diff
@@ -59,3 +63,42 @@ class ExtractLines(Aggregate):
 class ExtractPolys(Aggregate):
     name = "extracted_polys"
     template = "ST_CollectionExtract(ST_Collect(%(expressions)s), 3)"
+
+
+def reverse_geocode_and_store_address_geometry(permit_request, to_geocode_addresses):
+    geoadmin_address_search_api = (
+        "https://api3.geo.admin.ch/rest/services/api/SearchServer"
+    )
+    geom = GeometryCollection()
+    for address in to_geocode_addresses:
+
+        search_params = {
+            "searchText": address,
+            "limit": 1,
+            "partitionlimit": 1,
+            "type": "locations",
+            "sr": "2056",
+            "lang": "fr",
+            "origins": "address",
+        }
+
+        data = urllib.parse.urlencode(search_params)
+        url = f"{geoadmin_address_search_api}?{data}"
+        response = requests.get(url)
+        if response.status_code == 200 and len(response.json()["results"]) > 0:
```

Comment:
`if response.status_code == 200 and response.json()["results"]:` également possible afin d'éviter un `len`